streams/src/main/java/org/apache/kafka/streams/state/internals/WindowKeySchema.java

```diff
@@ -33,15 +39,59 @@ public void init(final String topic) {
 
     @Override
     public Bytes upperRange(final Bytes key, final long to) {
-        return WindowStoreUtils.toBinaryKey(key, to, Integer.MAX_VALUE, serdes);
+        final byte[] bytes = key.get();
```

Comment:
is this and `lowerRange` identical to those in `SessionKeySchema`? If so then maybe we should extract them to avoid code duplication?